### ISSUE TITLE
New fetures for script

### ISSUE THREAD

**Issue by forall** [dalgibbard/citrix_xenserver_patcher#54]
Hi,
It's possible to add script a option -> download only a patch without patching?

**Comment by dalgibbard** [comment]
Yes it's possible, but I'm not sure it holds much value. You could review the XML for the patches you want to download, as that has the download links in it, as a sort term option?

Once you've downloaded only, what's the next steps? ie. What's the use case/requirements for this?

**Comment by forall** [comment]
Sometimes the download patches take a long time.. so I can run script which download all patches and next day I can apply it.

**Comment by dalgibbard** [comment]
Gotcha, might be possible then. The only thing I'm wondering is how we verify that the local file is valid on the install; the download could have been interrupted for example...

**Comment by forall** [comment]
This is a good question....Maybe it's possible to check md5sum patches from xml file?

**Comment by dalgibbard** [comment]
Can't remember if they're listed or not. I'll look into it at some point anyway :)

**Comment by abien** [comment]
+1 for this. Reason:
We have a bunch of XenServers. Instead of running this patcher locally(/home/patcher) from each and every system, we (would like to) run it on multiple server at once from a central NFS share.
ex: /mnt/patcher 

However, currently this isnt working because the downloads will get messed up due to parallel access.

It would be good if the script would allow me to just keep files and reuse them instead of download everyfile on every run. Maybe even specify the download dir.
